discovery_engine/lib/src/ffi/types/engine.dart

```diff
@@ -420,24 +316,6 @@ class DiscoveryEngineFfi implements Engine {
     return resultVoidStringFfiAdapter.consumeNative(result);
   }
 
-  @override
-  Future<List<DocumentWithActiveData>> deepSearch(
```

Comment:
We don't use deep search anymore? It was used for a PoC, on an older app branch, but I would double check with @LeifXayn or @frankpepermans maybe?

Reply:
the deep search still exists, it was moved to `search_by_id()` for the rust db version, the old name `deep_search()` was kept for the dart db version. also the dart event names and arguments didn't change, so this shouldn't be a breaking change for the app.